frontend/public/templates/work_packages/query_filters.html

```diff
@@ -100,6 +104,7 @@
 
           <span class="inline-label" ng-switch-when="date_past">
             <input ng-model="filter.textValue"
+                   ng-model-options="{ updateOn: 'default blur', debounce: { 'default': 400, 'blur': 0 } }"
```

Comment:
As this keeps repeating itself, wouldn't it make sense to place it inside a method of the scope?